_episodes/21-carpentries.md

```diff
@@ -378,26 +378,26 @@ you are very welcome to put your name forward as a candidate.
 > What would you add, and what would you drop to make room?
 {: .challenge}
 
-[dc-blog]: {{ site.dc_site }}/blog/
+[dc-blog]: <{{ site.dc_site }}/blog/>
```

Comment:
It shouldn't... but it seems to anyway, without showing the URLs explicitly anywhere on the page, as far as I could tell. At least locally on my laptop (and sounds like on @lexnederbragt's as well). It doesn't work on your local setup?